modules/kibana/src/internalClusterTest/java/org/elasticsearch/kibana/KibanaThreadPoolIT.java

```diff
@@ -32,11 +33,11 @@ protected Collection<Class<? extends Plugin>> nodePlugins() {
     public void testKibanaThreadPool() {
         runWithBlockedThreadPools(() -> {
             // index documents
-            String idToDelete = client().prepareIndex(".kibana").setSource(Map.of("foo", "delete me!")).get().getId();
-            String idToUpdate = client().prepareIndex(".kibana").setSource(Map.of("foo", "update me!")).get().getId();
+            String idToDelete = ESIntegTestCase.client().prepareIndex(".kibana").setSource(Map.of("foo", "delete me!")).get().getId();
```

Comment:
This extra reference to `ESIntegTestCase` is a side-effect of the move refactor?

Reply:
Good catch -- it appeared in a refactor but isn't needed. I'll remove it.